src/components/NavBar.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <nav>
+    <div class="nav-container">
+      <ul class="links-container">
+        <h6>
+          <router-link to="/"
+            ><img src="@/assets/logo_green.png"
+          /></router-link>
+        </h6>
```

Comment:
move lines 5-9 outside links-container

```
    <div class="nav-container">
     <h6>
      <router-link to="/"
        ><img src="@/assets/logo_green.png"
      /></router-link>
     </h6>      
     <ul class="links-container">
 ...
```

will result in 
![image](https://user-images.githubusercontent.com/80747408/133480943-b83d98ee-e853-4191-9d5f-5ab8bca4ce20.png)